src/routes/docs/advanced/platform/+layout.svelte

```diff
@@ -14,6 +14,10 @@
                 {
                     label: 'Overview',
                     href: '/docs/advanced/platform'
+                },
+                {
+                    label: 'Keyboard Shortcuts',
```

Comment:
Let's make this "Shortcuts" as we want to try and use 1 word for navigation as much as possible. The title of the page can still be "Keyboard Shortcuts" for clarity.

```suggestion
                    label: 'Shortcuts',
```